src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
```

Comment:
Consider renaming the 'result' variable to 'invertedRobot' for better readability and understanding of the code.

Reply:
fixed